packages/amplify_storage_s3/lib/method_channel_storage_s3.dart

```diff
@@ -166,15 +168,23 @@ class AmplifyStorageS3MethodChannel extends AmplifyStorageS3 {
 
   StorageException _prepareExceptionForMalformedResult(
       {@required String methodName, @required String fieldName}) {
-    final Map errorDetails = <String, String>{
-      '$methodName operation failed': '$fieldName cannot be null'
+    const exceptionType =
```

Comment:
#TIL [_Const, Static, Final, Oh my!_](https://news.dartlang.org/2012/06/const-static-final-oh-my.html)

Reply:
And a const constructor

```dart
// Lots of const keywords here.
const pointAndLine = const {
  'point': const [const ImmutablePoint(0, 0)],
  'line': const [const ImmutablePoint(1, 10), const ImmutablePoint(-2, 11)],
};
```